linkerd/core/src/main/scala/io/buoyant/linkerd/FailureAccrualInitializer.scala

```diff
@@ -16,7 +16,7 @@ abstract class FailureAccrualConfig extends PolymorphicConfig {
   var backoff: Option[BackoffConfig] = None
 
   @JsonIgnore
-  val backoffOrDefault =
+  lazy val backoffOrDefault =
```

Comment:
Hmm. I used `lazy val` since the original code used a `val` and it seemed like the intention was for this value to not change after the config is parsed. Are we expecting that configs will ever be parsed?